cgi/import_photos_upload.pl

```diff
@@ -155,24 +155,24 @@
     </script>
 
     <!-- The Templates plugin is included to render the upload/download listings -->
-    <script src="/js/dist/tmpl.js"></script>
+    <script src="[% static_subdomain %]/js/dist/tmpl.js"></script>
```

Comment:
[% static_subdomain %] only works for templates that are processed by Template::Toolkit (files in /templates/).

In Perl code, you need to use $static_subdomain instead